keyboards/tkc/california/keymaps/via/keymap.c

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2021 Terry Mathews
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
VIA requires that all 4 layers are populated, or the config changed so that the number of layers is reduced.

Reply:
#define DYNAMIC_KEYMAP_LAYER_COUNT 2 is set in config.h. There's not enough space for four layers when I try compile, throws an error.

Reply:
I swear I double checked!  But yeah, as long as that's set, it's fine.